spec/definitions/Scheduling/Time.yaml

```diff
@@ -0,0 +1,2 @@
+type: string
+pattern: "^([0-9]|0[0-9]|1[0-9]|2[0-3]):[0-5][0-9]:[0-5][0-9](Z(\\+|\\-)([0-9]|0[0-9]|1[0-9]|2[0-3]):[0-5][0-9])?$"
```

Comment:
I asked @adamaltman for advice and I think we can go down to 1-hour resolution, so if you agree you can update this regex.